apps/common/components/SummaryData.tsx

```diff
@@ -0,0 +1,30 @@
+
+import type {ReactElement} from 'react';
+
+type TItem = {
+    label: string,
+    content: string
+}
+
+type TTabsProps = {
+	items: TItem[],
+}
+
+function SummaryData({items}: TTabsProps): ReactElement {
```

Comment:
I would just inline the type as it's just a single prop

```suggestion
function SummaryData({items}: {items: TItem[]}): ReactElement {
```